dart-templates/api.mustache

```diff
@@ -121,9 +122,7 @@ class {{classname}} {
             bodyData = {{paramName}}{{#isFile}}.finalize(){{/isFile}};
         {{/isPrimitiveType}}
         {{^isPrimitiveType}}
-            bodyData = {{^required}}{{paramName}} == null ? null : {{/required}}{{paramName}}.toJson();
-            // bodyData = {{^required}}{{paramName}} == null ? null : {{/required}}jsonEncode({{paramName}});
-            // bodyData = {{^required}}{{paramName}} == null ? null : {{/required}}jsonDecode(jsonEncode({{paramName}}));
+            bodyData = {{^required}}{{paramName}} == null ? null : {{/required}}{{paramName}}.toJson(bodyParameters);
```

Comment:
`bodyData = {{paramName}}{{^required}}?{{/required}}.toJson(bodyParameters);`
Would it work ?